docker-app/qfieldcloud/core/tests/test_team.py

```diff
@@ -0,0 +1,156 @@
+import logging
+from django.utils.timezone import now
+from qfieldcloud.authentication.models import AuthToken
+from qfieldcloud.core.models import (
+    Organization,
+    OrganizationMember,
+    Person,
+    Team,
+)
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .utils import set_subscription, setup_subscription_plans
+
+logging.disable(logging.CRITICAL)
+
+
+class TeamsTestCase(APITestCase):
+    def setUp(self):
+        setup_subscription_plans()
+
+        self.user1 = Person.objects.create_user(username="user1", password="abc123")
+        self.token1 = AuthToken.objects.get_or_create(user=self.user1)[0]
+
+        self.user2 = Person.objects.create_user(username="user2", password="abc123")
+        self.token2 = AuthToken.objects.get_or_create(user=self.user2)[0]
+
+        self.user3 = Person.objects.create_user(username="user3", password="abc123")
+        self.token3 = AuthToken.objects.get_or_create(user=self.user3)[0]
+
+        self.organization1 = Organization.objects.create(
+            username="organization1",
+            password="abc123",
+            type=Organization.Type.ORGANIZATION,
+            organization_owner=self.user1,
+        )
+
+        # Activate Subscriptions
+        set_subscription(self.organization1, "default_org")
+
+        OrganizationMember.objects.create(
+            organization=self.organization1,
+            member=self.user2,
+            role=OrganizationMember.Roles.ADMIN,
+        )
+        OrganizationMember.objects.create(
+            organization=self.organization1,
+            member=self.user3,
+            role=OrganizationMember.Roles.MEMBER,
+        )
+
+    def test_list_teams(self):
+        team1 = Team.objects.create(
+            username=f"@{self.organization1.username}/team1",
+            team_organization=self.organization1,
+        )
+        team2 = Team.objects.create(
+            username=f"@{self.organization1.username}/team2",
+            team_organization=self.organization1,
+        )
+
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token1.key)
+        response = self.client.get(
+            f"/organizations/{self.organization1.username}/team/"
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        json_response = response.json()
+        team_usernames = [team["username"] for team in json_response]
+
+        self.assertIn(team1.username, team_usernames)
+        self.assertIn(team2.username, team_usernames)
```

Comment:
```suggestion
        payload = response.json()
        team_usernames = [team["username"] for team in payload]

        self.assertIn(team1.username, team_usernames)
        self.assertIn(team2.username, team_usernames)
```

What if the json_response is not sorted in the way you expected?

Reply:
Changed!